### PR TITLE
managing-vm-kernel 3.2 updates

### PR DESCRIPTION
**Pull request by awokd**
Committing what I have so far, so I don't lose my place. This is current for R3.2.
R4.0 steps/verification has a couple pending issues, will back-link here for tracking. Suggest merging this now if it looks good, then coming back to address R4.0 once those issues are resolved.

### CODE REVIEW

File: configuration/managing-vm-kernel.md [hunk 1]
```diff
@@ -130,21 +135,22 @@ Complete!
 [marmarek@dom0 ~]$
```
Comment (marmarek) [inline configuration/managing-vm-kernel.md]: Oh, username have changed from one shell prompt to another ;)

File: configuration/managing-vm-kernel.md [hunk 2]
```diff
@@ -58,10 +63,10 @@ kernel-qubes-vm-3.18.16-3.pvops.qubes.x86_64
 kernel-qubes-vm-3.18.17-4.pvops.qubes.x86_64
 ~~~
 
-If you want a more recent version, you can check `qubes-dom0-unstable` repository. As the name suggests, keep in
-mind that those packages may be less stable than the default ones.
+If you want a more recent version, you can check the `qubes-dom0-unstable` repository.
+As the name suggests, keep in mind that those packages may be less stable than the default ones.
```
Comment (marmarek) [inline configuration/managing-vm-kernel.md]: There is also `kernel-latest-qubes-vm` package - which should provide more recent (non-LTS) kernel, but much less tested.

File: configuration/managing-vm-kernel.md [hunk 3]
```diff
@@ -259,29 +271,34 @@ You can safely ignore this error message:
...
 ~~~
-sudo apt-get update
-sudo apt-get install qubes-kernel-vm-support grub2-common
+sudo apt update
+sudo apt install qubes-kernel-vm-support grub2-common
 ~~~
 
+If prompted for a GRUB install device, choose `/dev/mapper/dmroot`.
+You will receive an error about GRUB failed to install to it, but just continue anyways.
+You don't need an actual grub bootloader as it is provided by dom0, but having one shouldn't hurt.
```
Comment (marmarek) [inline configuration/managing-vm-kernel.md]: This sentence is duplicated.

**Comment by awokd** [comment]
Updated.

**Pull request merged**